flags/README.md

```diff
@@ -6,7 +6,7 @@ Command line arguments parser for Deno based on minimist
 
 ``` ts
 import { args } from "deno";
-import parseArgs from "https://deno.land/x/parseargs/index.ts";
+import parseArgs from "https://deno.land/x/flags/index.ts";
```

Comment:
import flags ?

Reply:
It's an example how to import this module in your code.

Reply:
It think it would be better to just use “flags” for the namespace here too

Reply:
Do you mean `import { flags } from "https://deno.land/x/flags/index.ts";`?

Reply:
import * as flags from “...flags.ts”;

Would be the preferable form. I just realized that this lib has a default export. Default exports are not the preferred style afaik ... so maybe rename that function to “parse” so it might be 
import { parse } from “....flags”;

(Sorry for formatting - on my phone)

Reply:
Let's just fix the links for now. LGTM